### PR TITLE
Fix incorrect type for follower group in profile

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Noticed this was typed incorrectly, fixed the type and added some more specific types elsewhere

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-follow-type

File: packages/common/src/store/pages/profile/actions.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function updateCurrentUserFollows(follow = false, handle: string) {
 }
 
 export function fetchFollowUsers(
-  followerGroup: User[],
```
Comment (Kyle-Shanks) [inline packages/common/src/store/pages/profile/actions.ts:91]: Was this just very wrong before or did something else change to make the type change?

Comment (rickyrombo) [reply packages/common/src/store/pages/profile/actions.ts:91]: Just very wrong! Most usages are still in Javascript, and the one Typescript one was using `any`

Comment (Kyle-Shanks) [reply packages/common/src/store/pages/profile/actions.ts:91]: gotcha, makes sense

File: packages/web/src/pages/profile-page/ProfilePageProvider.tsx [hunk 2]
```diff
@@ -1040,7 +1040,11 @@ function mapDispatchToProps(dispatch: Dispatch, props: RouteComponentProps) {
     playUserFeedTrack: (uid: UID) => dispatch(feedActions.play(uid)),
     pauseUserFeedTrack: () => dispatch(feedActions.pause()),
     // Followes
-    fetchFollowUsers: (followGroup: any, limit: number, offset: number) =>
+    fetchFollowUsers: (
+      followGroup: FollowType,
```
Comment (Kyle-Shanks) [inline packages/web/src/pages/profile-page/ProfilePageProvider.tsx:1044]: nit: followGroup instead of followerGroup make cause grepping confusion

**Review by Kyle-Shanks — APPROVED** [review]
Nice 👍

**Pull request merged**